go.mod

```diff
@@ -2,33 +2,28 @@ module github.com/openshift/windows-machine-config-operator
 
 go 1.16
 
-replace (
```

Comment:
I suggest naming this commit `[vendor] Update openshift/api and openshift/client-go`.